README.md

```diff
@@ -50,7 +52,8 @@ add her to the list of guests in the castle.
 
 Write your code here:
 ```ruby
-# code here
+town[residents].delete(2)
+town[castle][:guests] = "Belle"
 ```
 
```

Comment:
we went over this, so yes, again think about how to drill down in objects and be more mindful of your syntax. you're close here.